frontend/resourceadm/components/LeftNavigationBar/LeftNavigationBar.module.css

```diff
@@ -37,35 +19,60 @@
   padding-left: 15px;
   padding-block: 20px;
   border: none;
-  border-bottom: 1px solid #bebebe;
+  border-bottom: 1px solid var(--fds-semantic-border-neutral-divider);
   background-color: transparent;
   cursor: pointer;
 }
 
-.navigationElementSelected {
-  display: flex;
-  align-items: center;
-  padding-left: 15px;
-  padding-block: 20px;
-  border: none;
-  border-bottom: 1px solid #bebebe;
-  background-color: #E3F7FF;
-  cursor: pointer;
+.backButton {
+  border-bottom: 2px solid var(--fds-semantic-border-neutral-divider);
+}
+
+.buttonText {
+  color: inherit;
+}
+
+.navigationElement:hover {
+  background-color: var(--fds-semantic-background-default);
+  z-index: 1;
+}
+
+.backButton:hover {
+  background-color: var(--fds-semantic-surface-neutral-subtle-hover);
+}
+
+.navigationElement:focus-within,
+.navigationElement:focus-visible {
+  --fds-inner-focus-border-color: var(--fds-semantic-border-focus-boxshadow);
+  --fds-outer-focus-border-color: var(--fds-semantic-border-focus-outline);
+  --fds-focus-border-width: 3px;
+
+  border-bottom: 1px solid var(--fds-semantic-border-neutral-divider);
+  background-color: var(--fds-semantic-background-default);
+  outline: var(--fds-focus-border-width) solid var(--fds-outer-focus-border-color);
+  outline-offset: var(--fds-focus-border-width);
+  box-shadow: 0 0 0 var(--fds-focus-border-width) var(--fds-inner-focus-border-color);
+  z-index: 2;
```

Comment:
Because the "buttons" used in the navigation bar are directly next to each other. To ensure that the borders are still shown as inteded, this was done as a hack :) Discussed and agreed with one from the design system.